int32 tracking_no
float32 pos_x
float32 pos_y
float32 pos_z
float32 distance
